### ISSUE TITLE
question about annealing

### ISSUE THREAD

**Issue by VincenzoUrso** [akumidv/tradingview-assistant-chrome-extension#91]
I'd like to know how to best make use of the annealing method. In particular, my question is:
as annealing make use of the results to find the best values, if I do an X iteration cycle, the cycle ends, I do small changes in the values and restart another cycle without clearing the extension memory, are the previous cycle values still relevant for the new one?

In other words, is the work done in an X cycle + Y cycle the "same" as a 2X cycle?

Any other relevant information will be appreciated.
Thanks

**Comment by akumidv** [comment]
You are right, the two cycles of annealing method do not the same as twice increased number of cycles.
The annealing method on first part of cycle is trying to find new extremum in area of parameters hyperspace. On the second part it try to improve values of had found area

https://en.wikipedia.org/wiki/Simulated_annealing

[![Annealing method](image)](https://en.wikipedia.org/wiki/File:Hill_Climbing_with_Simulated_Annealing.gif)

**Comment by VincenzoUrso** [comment]
Ok thanks. It would be nice if annealing can avoid replicate the same values as it sometimes do

**Comment by akumidv** [comment]
Ok. I'll think how it possible to implement
